packages/cohort/src/bin/replicator.rs

```diff
@@ -30,10 +39,14 @@ async fn main() {
     kafka_consumer.subscribe().await.unwrap();
 
     //  c. Create suffix.
+    let replicator_config = ReplicatorServiceConfig {
+        commit_frequency_ms: env_var_with_defaults!("REPLICATOR_KAFKA_COMMIT_FREQ_MS", u64, 10_000),
```

Comment:
Having some re-usable function which makes configs from env might be useful. I think I saw similar code in another file.